src/ShipmentRequest.php

```diff
@@ -36,6 +36,11 @@ class ShipmentRequest extends QuoteRequest
      */
     public $signatureRequired = false;
 
+    /**
+     * @var string
+     */
+    public $itn;
```

Comment:
Please avoid abbreviations for all names that are not immediately obvious to the reader like `$i, $x, $y, $no` etc. This should probably be called `$internationalTransactionNo`.